training/unit_tests.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+
+from rlbot.training.training import Pass, Fail
+from rlbottraining.exercise_runner import run_playlist
+
+from hello_world_training import StrikerPatience
+
+class CommonExercisesTest(unittest.TestCase):
```

Comment:
Please document clearly that this is a test for the tests, and not something that most people would care about running unless they're troubleshooting.

Reply:
I think the class name here is wrong.
This is not a test for tests. This is an example for how to test that your bot works as expected.
I think people should care to test their bots for regression test purposes.

Reply:
Oh, my mistake. I'm still interested in appropriate docs, easy enough to get fooled apparently :)